AEPMessaging/Tests/UnitTests/SurfaceTests.swift

```diff
@@ -17,9 +17,61 @@ import XCTest
 import AEPServices
 
 class SurfaceTests: XCTestCase {
-                
-    override func setUp() {
+    
+    let mockAppSurface = "mobileapp://com.apple.dt.xctest.tool"
+    
+    func testSurfaceHappy() {
+        // test
+        let surface = Surface(path: "myTestView")
+        
+        // verify
+        XCTAssertEqual("\(mockAppSurface)/myTestView", surface.uri)
+        XCTAssertTrue(surface.isValid)
+    }
+    
+    func testSurfaceEmptyPath() {
+        // test
+        let surface = Surface(path: "")
+        
+        // verify
+        XCTAssertEqual("", surface.uri)
```

Comment:
is this correct behavior?  wouldn't an empty path just give us the base without a path?

Reply:
As discussed offline, we will change this behavior to have `Surface(path: "")` and `Surface()` behave the same.